samples/multiplatform/androidApp/build.gradle

```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = '1.2.31'
```

Comment:
it `1.2.31` required? can we use latest `1.2.41`?

Reply:
Done